scriptworker/gpg.py

```diff
@@ -1036,18 +1055,18 @@ def rebuild_gpg_home_flat(context, real_gpg_home, my_pub_key_path, my_sec_key_pa
         # sign all the keys
         for fingerprint in fingerprints:
             log.info("signing {} with {}".format(fingerprint, my_fingerprint))
-            sign_key(
+            await sign_key(
                 context, fingerprint, signing_key=my_fingerprint,
                 gpg_home=tmp_gpg_home
             )
         # Copy tmp_gpg_home/* to real_gpg_home/* before nuking tmp_gpg_home/
         overwrite_gpg_home(tmp_gpg_home, real_gpg_home)
 
 
-def rebuild_gpg_home_signed(context, real_gpg_home, my_pub_key_path,
-                            my_sec_key_path, trusted_path, untrusted_path=None,
-                            ignore_suffixes=(),
-                            consume_function=consume_valid_keys):
+async def rebuild_gpg_home_signed(context, real_gpg_home, my_pub_key_path,
+                                  my_priv_key_path, trusted_path,
+                                  untrusted_path=None, ignore_suffixes=(),
+                                  consume_function=consume_valid_keys):
     """Rebuild `real_gpg_home` with new trustdb, pub+secrings, gpg.conf.
 
     In this 'signed' model, import all the pubkeys in `consume_path` and sign
```

Comment:
Probably a bad copy and paste: `consume_path` doesn't exist and `rebuild_gpg_home_signed()` does make some fingerprints trusted.